src/solver.jl

```diff
@@ -1,11 +1,11 @@
-immutable GIValue{G <: AbstractGrid}
+struct GIValue{G <: AbstractGrid}
     grid::G
     gdata::Vector{Float64}
 end
 
 evaluate(v::GIValue, s::AbstractVector{Float64}) = interpolate(v.grid, v.gdata, convert(Vector{Float64}, s))
 
-@with_kw type CWorldSolver{G<:AbstractGrid, RNG<:AbstractRNG} <: Solver
+@with_kw struct CWorldSolver{G<:AbstractGrid, RNG<:AbstractRNG} <: Solver
```

Comment:
Probably `mutable struct`

Reply:
Yeah that was causing an error in testing.